icon_validator/rules/plugin_validators/runtime_validator.py

```diff
@@ -0,0 +1,56 @@
+import os
+import glob
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class RuntimeValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_setup(spec):
+        if "setup.py" in os.listdir(spec.directory):
+            with open(f"{spec.directory}/setup.py", "r") as file:
+                setup_str = file.read().replace("\n", "")
+
+                if "komand" in setup_str and "insightconnect-plugin-runtime" not in setup_str:
+                    raise ValidationException("Komand is no longer used in setup.py. "
```

Comment:
We need to update this error message. It sounds like we need to update all the `komand` words in setup.py with `insightconnect-plugin-runtime`.
 We have komand word in setup.py as  `scripts=['bin/komand_plugin_name']`. @zyoutz-r7  Are we expecting this dir-name to be updated? If yes, then I have to update the `icon-plugin` as well.

Reply:
Good catch! We will only want to match on `komand` being part of `install_requires`.  The name of the plugin which impacts the `scripts` value could still have komand in it and not be a problem.

Reply:
If possible, probably best to try to only extract what is between:
```
install_requires=[
        "insightconnect-plugin-runtime"
    ]
```
Anything else could potentially have komand in it and not be a problem.